CHANGELOG.md

```diff
@@ -87,6 +87,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We fixed an issue where the "Import by ID" function would throw an error when a DOI that contains URL-encoded characters was entered. [#10648](https://github.com/JabRef/jabref/issues/10648)
 - We fixed an issue with handling of an "overflow" of authors at `[authIniN]`. [#11087](https://github.com/JabRef/jabref/issues/11087)
 - We fixed an issue where an exception occurred when selecting entries in the web search results. [#11081](https://github.com/JabRef/jabref/issues/11081)
+- We fixed a bug where the "Directory not set" warning would come up multiple times when importing multiple entries without saving library, and would not come up at all for some fetchers if there was no pdf associated. [#11075](https://github.com/JabRef/jabref/issues/11075)
```

Comment:
Uff, too long text

- When a new library is unsaved, there is now a warning when fetching PDFs.

Reply:
Changed 😅

Reply:
But Oliver, wasn't the warning already there? The fix addresses the case of multiple warnings (in case of pdfs) and no warning (in case of no pdfs).

Reply:
Then use "improved warning"

Not needed to describe the details since all of this is an edge case!